modin/tests/pandas/dataframe/test_map_metadata.py

```diff
@@ -223,13 +223,51 @@ def test_add_suffix(data, axis):
     df_equals(new_modin_df.columns, new_pandas_df.columns)
 
 
+map_strtagies = ["map", "axis_map", "splitted_axis_map"]
+
+
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 @pytest.mark.parametrize("testfunc", test_func_values, ids=test_func_keys)
 @pytest.mark.parametrize(
     "na_action", [None, "ignore"], ids=["no_na_action", "ignore_na"]
 )
-def test_applymap(data, testfunc, na_action):
+@pytest.mark.parametrize("map_strtagy", map_strtagies, ids=map_strtagies)
+def test_applymap(data, testfunc, na_action, map_strtagy):
+    keys = list(data.keys())
+    epected_shape = None
+    if map_strtagy == "map":
+        epected_shape = (1, 1)
+        max_size = max(len(keys), len(data[keys[0]]))
+        MinPartitionSize.put(max_size)
```

Comment:
Calling `put` function (for all config variables) without returning the old value may affect other tests. I suggest using `context` function from the same module.

Reply:
Outdated since the test was moved to test_internals.